Fix CI multibox configuration and Show multibox information

unakatsuo opened the pull request:
Found  several incomplete configurations in ci/mulitibox.
- Running package install steps on the nodes which don't need to.
- Duplicate hostname
- Fix wrong ZK address in ``/etc/openvdc/executor.toml``

This branch can get green status only when ``REBUILD`` Jenkins flag is true currently. 

During the CI debugging, I added new features to get information from all multibox nodes:
- Run ``systemctl status`` before running acceptance tests.
- Show ``journalctl`` log when acceptance tests finished.

Commit:
ci: Wait for listen port of API server becomes ready.

Commit:
ci: Dump logs from all nodes after test complete.

Commit:
Revert "ci: Fix missing executable bit"

This reverts commit 123521eef2a0826fd69eb98f2eaf2b1ed760d11a.

Commit:
ci: Fix missing executable bit

Commit:
ci: Recover symlink from normal file. Caused by d03fab7d9b.

Commit:
ci: Allow ssh to pass optional parameters.

Commit:
ci: Add missing configuration.

Commit:
ci: Update /etc/hosts for all nodes.

Commit:
ci: Run systemctl status on all nodes.

Commit:
ci: Insert log section title with node name.

Commit:
ci: Disable mesos to lookup hostname.

unakatsuo commented:
https://ci.openvdc.org/job/citest/job/fix-ci-config/3/consoleFull

This job was kicked with ``REBUILD=false`` after the success build with ``REBUILD=true``. It ran ``Copy cache from **** branch`` step for master branch. Base VM images seem to be copied from master.

```
** DOING STEP: Copy cache from master branch
sending incremental file list
./
executor-lxc.qcow2
executor-null.qcow2
mesos.qcow2
scheduler.qcow2
sshkey_executor-lxc
sshkey_executor-lxc.pub
sshkey_executor-null
sshkey_executor-null.pub
sshkey_mesos
sshkey_mesos.pub
sshkey_scheduler
sshkey_scheduler.pub
sshkey_zookeeper
sshkey_zookeeper.pub
zookeeper.qcow2

sent 4034578446 bytes  received 300 bytes  206901474.15 bytes/sec
total size is 4034084998  speedup is 1.00
```

build.env

```
+ . ./build.env
++ LEAVE_CONTAINER=0
++ REPO_BASE_DIR=
++ BUILD_CACHE_DIR=
++ BUILD_OS=el7
++ REBUILD=false
++ RELEASE_SUFFIX=20170213095902gitd93a82f
++ GIT_BRANCH=fix-ci-config
++ BRANCH_NAME=fix-ci-config
++ BRANCH=fix-ci-config
++ SHA=d93a82fa16c8613f8bc26b7705fa4b569643549b
```

Metallion commented on ci/acceptance-test/multibox/10.0.100.14-vdc-executor-lxc/guestroot/etc/openvdc/executor.toml:
```diff
@@ -2,4 +2,4 @@
 
 [zookeeper]
-endpoint = "zk://localhost/openvdc"
+endpoint = "zk://10.0.100.10/openvdc"
```
My experience was that this line gets ignored which is why I didn't fill it in. I'd like to discuss a bit what exactly is going on here before merge.

Metallion commented on ci/acceptance-test/multibox/ind-steps/step-hosts/install.sh:
```diff
@@ -0,0 +1,14 @@
+10.0.100.11 mesos
+10.0.100.12 scheduler
+10.0.100.13 executor-null
+10.0.100.14 executor-lxc
```
I still don't understand why this is needed since I have seen tests work without it and we can't go around hard coding names. I'd like to discuss this a bit too.

Commit:
ci: Re-use VM images for topic branch if available.

It was not intended that the base VM images from BASE_BRANCH were copied always when REBUILD=false.

Commit:
ci: Wait for instance to become RUNNING.

Commit:
build: Build acceptance test binary from build.go

Commit:
ci: Define exec.Cmd object in loop.

Commit:
ci: Configure daemon enable/disable flag before booting VM.

The framework received mesos-slave lost event occasionally.  The issue is that vdc-scheduler dies at the event since its cluster change event handler  is not implemented properly yet.

Commit:
ci: Fix daemon name

unakatsuo commented:
Suspicious log in ``mesos`` (10.0.100.11) node.

```
Feb 14 20:26:34 mesos systemd[1]: Stopping Mesos Slave...
Feb 14 20:26:34 mesos mesos-slave[736]: W0214 20:26:33.757333   689 logging.cpp:91] RAW: Received signal SIGTERM from process 1 of user 0; exiting
Feb 14 20:26:34 mesos mesos-master[742]: I0214 20:26:34.252440   926 master.cpp:1258] Agent 0672afa8-07d6-4caf-b0aa-1bc84f6bbeec-S0 at slave(1)@10.0.100.11:5051 (mesos) disconnected
Feb 14 20:26:34 mesos mesos-master[742]: I0214 20:26:34.252497   926 master.cpp:2961] Disconnecting agent 0672afa8-07d6-4caf-b0aa-1bc84f6bbeec-S0 at slave(1)@10.0.100.11:5051 (mesos)
Feb 14 20:26:34 mesos mesos-master[742]: I0214 20:26:34.252528   926 master.cpp:2980] Deactivating agent 0672afa8-07d6-4caf-b0aa-1bc84f6bbeec-S0 at slave(1)@10.0.100.11:5051 (mesos)
Feb 14 20:26:34 mesos mesos-master[742]: I0214 20:26:34.252631   926 hierarchical.cpp:584] Agent 0672afa8-07d6-4caf-b0aa-1bc84f6bbeec-S0 deactivated
Feb 14 20:26:34 mesos systemd[1]: Stopped Mesos Slave.
```
